### DISCUSSION TITLE
Add zulip or slack channel for Polaris

### DISCUSSION THREAD

**Discussion by jbonofre** [apache/polaris#14] [Ideas]
In order to facilitate communication, mentor contributors and expend our community, it would be great to have a instant chat platform like Zulip or Slack.

I would be happy to tackle this out.

Thoughts ?

@snazy @annafil @takidau @RussellSpitzer

**Comment by snazy** [comment]
+1 - I'd be happy to set it up quickly :)

**Reply by jbonofre** [reply]
Cool ! Zulip I guess ?

**Reply by snazy** [reply]
I'd prefer Zulip - it's "channel / topic" approach is really neat, we made only positive experiences w/ Zulip for Nessie.

**Comment by annafil** [comment]
✨ Sounds great! Should we propose that this live as a "polaris-catalog" channel inside the Iceberg slack or are you thinking more of a standalone chat community? My preference is for the former to help folks avoid switching contexts but not a super strongly held opinion :)

**Reply by snazy** [reply]
As a separate chat, because it's a separate project.

**Reply by annafil** [reply]
Roger that!

**Comment by MonkeyCanCode** [comment]
Do we have one created?

**Reply by dimas-b** [reply]
https://polaris-catalog.zulipchat.com

**Comment by takidau** [comment]
Sorry to come to this late. Zulip appears to not be approved by Snowflake IT. Looks like we can probably use our personal accounts to join, or I could work on getting Zulip approved by IT. Or we could consider switching to Slack, which I feel probably has broader adoption than Zulip. But I don't want to rock the boat either if folks are already using Zulip heavily. What would be the right path forward, @snazy and @jbonofre ?
